index.js

```diff
@@ -36,7 +36,7 @@ const SCHEMA_SLACK_SETTINGS = Joi.object().keys({
             minimized: Joi.boolean()
         }),
         SCHEMA_SLACK_CHANNELS, SCHEMA_SLACK_CHANNEL
-    )
+    ).required()
```

Comment:
Is this `required()` needed?

Reply:
I used notifications-email as a reference, I think it would be better to have.
https://github.com/screwdriver-cd/notifications-email/blob/6a59ef146c698ce3e3f00acd2b4b520d64d2eef8/index.js#L32